consensus/types/actor_mapper_test.go

```diff
@@ -0,0 +1,190 @@
+package types
```

Comment:
This is awesome. Not only are the tests well defined and extensive, it also makes it very clear what all the mappers do.